### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by RedLikeRosesss**
Fixed discrepancy between the report page (Live Statistics tab) and the doc section describing it

### CODE REVIEW

File: docs/user-guide/out-of-resources/timeout.md [hunk 1]
```diff
@@ -483,7 +483,7 @@ reduce the load on these mappings by splitting them into smaller pieces,
 (smaller mappings or scalar variables), but this is not always possible and some
 mappings usually remain in the final SMT formula.
 
-Under this model, the "#total updates" is a measure of how many times we store
+Under this model, the "memory updates" is a measure of how many times we store
```
Comment (alexandernutz) [inline docs/user-guide/out-of-resources/timeout.md:486]: I'd drop the "the"

**Review by alexandernutz — APPROVED** [review]
great, thanks a lot for making this better!

File: docs/user-guide/out-of-resources/timeout.md (hunk 1 above)
Comment (RedLikeRosesss) [reply docs/user-guide/out-of-resources/timeout.md]: Done

**Review by shoham-certora — APPROVED** [review]
Looks good, approved!

**Pull request merged**